### PR TITLE
Wrapper script for cowpy

### PR DESCRIPTION
**Pull request by HemanthJabalpuri**
Will work for files size less than the destination.

### CODE REVIEW

**Comment by HemanthJabalpuri** [comment]
@trufae
can you please check this.

File: cowpy.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/system/bin/sh
+if [ "$#" -ne 2 ]; then
+  echo "Usage : cowpy.sh [src] [dst]"
+  exit 1
+elif ! [ -r "$1" ] || ! [ -r "$2" ]; then
+  echo " Files are not readable/not exist"
```
Comment (trufae) [inline cowpy.sh]: Prefix this echo with "Error: " and > /dev/stderr

**Review by trufae — APPROVED** [review]
lgtm

**Comment by trufae** [comment]
thanks!

**Comment by HemanthJabalpuri** [comment]
> thanks!

Sir, Is it safe to append nulls to get desired size?

**Comment by trufae** [comment]
truncating the file is the only way to change its size. appending zeros
will just be fine depending on the file format

**Pull request merged**